spec/features/budgets/votes_spec.rb

```diff
@@ -45,7 +45,7 @@
         visit budget_investments_path(budget, heading_id: heading.id)
 
         within('.supports') do
-          find('.in-favor a').click
+          accept_confirm { find('.in-favor a').click }
```

Comment:
This is not needed here as it makes the test fail

Reply:
Why can't both codebases be equal? Is this going to also fail on other forks as well?

Reply:
Yes... this is something pending to figure out...

The first time a user Supports an Investment, he sees a confirmation popup, which for some reason it wasn't required in tests...

We'll figure it out 😌